src/Entity/MajorVersion.php

```diff
@@ -282,33 +272,40 @@ public function toArray(): array
 
     public function isActive(): bool
     {
-        $dateTime = new \DateTimeImmutable();
-        return null === $this->getMaintainedUntil()
-            || $dateTime <= $this->getMaintainedUntil();
+        $dateTimeImmutable = new \DateTimeImmutable();
+        if (null === $this->getMaintainedUntil()) {
+            return true;
+        }
+        return $dateTimeImmutable <= $this->getMaintainedUntil();
     }
 
     public function isElts(): bool
     {
-        $dateTime = new \DateTimeImmutable();
-        return $this->getMaintainedUntil() != null
-            && $this->getEltsUntil() != null
-            && $dateTime > $this->getMaintainedUntil()
-            && $dateTime <= $this->getEltsUntil();
+        $dateTimeImmutable = new \DateTimeImmutable();
+        if ($this->getMaintainedUntil() == null) {
+            return false;
+        }
+        if ($this->getEltsUntil() == null) {
+            return false;
+        }
+        if ($dateTimeImmutable <= $this->getMaintainedUntil()) {
+            return false;
+        }
+        return $dateTimeImmutable <= $this->getEltsUntil();
```

Comment:
Disagree with this, too.

Reply:
It's the same as above. I tend to revert it...